### DISCUSSION TITLE
Chapter select before pushing play

### DISCUSSION THREAD

**Discussion by Jozodr** [advplyr/audiobookshelf-app#540] [Ideas]
Would it be possible to implement feature that would allow showing of chapters before pushing play for a book?
My use case: i have kids audio book with bunch of stories, each story is seperate file/chapter. Instead of me every time starting where I left off I would like to select specific chapter to start playing. 
Any thoughts?

**Comment by hobesman** [comment]
I like this idea. I believe it's been mentioned already but I don't remember the context. We listen to books falling asleep, and we like to jump to a specific chapter to start since that's what we remember as a marker while falling asleep. As in, "I remember starting chapter 5 before falling asleep..." so I want to start playing there.

**Reply by Jozodr** [reply]
Exactly, glad to see positive comment and that its in line with your use case. I believe it would improve usability.
Would this feature be worthy of project backlog?

**Comment by advplyr** [comment]
My thought was to add a list of the chapters somewhere on the item page like they are on the website. Then you can click the timestamp of the chapter to start playback.

I'm not sure this will be much more efficient then pressing play and opening the chapter menu in the player to select the chapter. It would be one less click.

Something similar was brought up before to have a menu pop up when you press play that allows you to select a chapter before playing. I didn't like this because it adds friction when most people just want it to play the audiobook where they last left off, or from the beginning.

**Comment by Jozodr** [comment]
I dont think this should be implemented as additional step when clicking play, it would be nice to have a chapter button next to it that would pop up as its already implemented in playing screen. The annoyance here is that to get to the specific chapter you have to start the book, you get the audio and it will play till you decide which chapter to go for.
